feat: 🎸 Update default to use chain v5.0.3

polymath-eric opened the pull request:
### Description

Removes support for 4.x versions, as they were broken due to the images being removed from docker hub. Also updates the default REST API to v0.1.0

### Breaking Changes

- chain version 4 is no longer supported

### JIRA Link

[DA-428](https://polymesh.atlassian.net/browse/DA-428)

### Checklist

- [ ] Updated the Readme.md (if required) ?

polymath-eric commented on src/common/uis.ts:
```diff
@@ -69,42 +81,11 @@ function haveUIs(): boolean {
- * Uses the REST API since the official S3 client requires credentials
- * @returns The S3 filename of the most recent UI for the given version
- */
-async function fetchFileName(version: string): Promise<Record<string, unknown> | void> {
```
In hindsight, S3 isn't great for this kind of stuff.

adamdossa commented:
Not directly related to this PR, but https://github.com/PolymeshAssociation/polymesh-local/blob/master/scripts/build-uis.ts#L21 can be updated to the `PolymeshAssociation` repo https://github.com/PolymeshAssociation/governance-web.

polymath-eric commented:
Thanks @adamdossa . I updated that reference, removed the deprecated upload script, and added a blurb in the README to document the process a bit better.

prashantasdeveloper commented:
:tada: This PR is included in version 5.0.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@polymeshassociation/polymesh-local/v/5.0.0)
- [GitHub release](https://github.com/PolymeshAssociation/polymesh-local/releases/tag/v5.0.0)
- `v5.0.0`
- `v5.0.0`

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
